contrib/win32/openssh/install-sshd.ps1

```diff
@@ -274,13 +274,25 @@ sc.exe privs sshd SeAssignPrimaryTokenPrivilege
 Add-Privilege -Account $sshdSid -Privilege SeAssignPrimaryTokenPrivilege
 Add-Privilege -Account $sshdSid -Privilege SeServiceLogonRight
 
+# create logs folder and set its permissions
 if(-not (test-path $logsdir -PathType Container))
 {
     $null = New-Item $logsdir -ItemType Directory -Force -ErrorAction Stop
 }
-$rights = [System.Security.AccessControl.FileSystemRights]"Read, Write"
-$accessRule = New-Object System.Security.AccessControl.FileSystemAccessRule($sshdAccount, $rights, "ContainerInherit,ObjectInherit", "None", "Allow")
 $acl = Get-Acl -Path $logsdir
-$Acl.SetAccessRule($accessRule)
+$acl.SetSecurityDescriptorSddlForm("O:BAG:DUD:PAI(A;OICI;FA;;;SY)(A;OICI;FA;;;BA)")
 Set-Acl -Path $logsdir -AclObject $acl
+
+$agentlog = Join-Path $logsdir "ssh-agent.log"
+if(-not (test-path $agentlog)){ $null | Set-Content $agentlog }
+Set-Acl -Path $agentlog -AclObject $acl
+
+$sshdlog = Join-Path $logsdir "sshd.log"
+if(-not (test-path $sshdlog)){ $null | Set-Content $sshdlog }
+Set-Acl -Path $sshdlog -AclObject $acl
+$rights = [System.Security.AccessControl.FileSystemRights]"Read, Write"
```

Comment:
who has the delete permissions?

Can you test downloading the log files using scp/sftp, just to make sure nothing is broken.

Reply:
Added SDDL description. These files are owned by Administrators group. Any administrator can delete them otherwise. IT should not make any other difference otherwise. scp and sftp work fine.